strr-web/pages/registry-dashboard.vue

```diff
@@ -207,35 +219,51 @@ const updateTableRows = async () => {
   const applications = await getPaginatedApplications(paginationObject)
   if (applications) {
     totalResults.value = applications.total
-    tableRows.value = registrationsToTableRows(applications)
+    tableRows.value = await registrationsToTableRows(applications)
   }
 
   updateMaxPageResults()
   loading.value = false
 }
 
-const registrationsToTableRows = (applications: PaginatedApplicationsI): Record<string, string>[] => {
+const registrationsToTableRows = async (applications: PaginatedApplicationsI): Promise<Record<string, string>[]> => {
   const rows: Record<string, string>[] = []
-  applications.applications.forEach((application: ApplicationI) => {
-    const { header, registration: { unitAddress, primaryContact } } = application
+  for (const application of applications.applications) {
+    const { header, registration: { primaryContact, registrationType } } = application
+    const contactPerson = primaryContact?.name
+    let certificateIssued = false
+    if (header.registrationStatus === RegistrationStatusE.ACTIVE && header.registrationId) {
+      certificateIssued = await isCertificateIssued(header.registrationId.toString())
+    }
 
     const row = {
       applicationNumber: header.applicationNumber,
-      location: unitAddress.city,
-      address: unitAddress.address,
-      owner: `
-        ${primaryContact.name.firstName}
-        ${primaryContact.name.middleName ?? ''}
-        ${primaryContact.name.lastName}
-      `,
+      registrationNumber: header.registrationNumber ?? '-',
+      registrationId: header.registrationId ? header.registrationId.toString() : '',
+      isCertificateIssued: certificateIssued,
+      registrationType: getRegistrationTypeLabel(registrationType),
+      contact: contactPerson
+        ? `${contactPerson.firstName} ${contactPerson.middleName ?? ''} ${contactPerson.lastName}`.trim()
+        : '-',
       status: header.examinerStatus || header.status,
       submissionDate: header.applicationDateTime
     }
     rows.push(row)
-  })
+  }
   return rows
 }
 
+const getRegistrationTypeLabel = (type: RegistrationTypeE): string => {
+  switch (type) {
+    case RegistrationTypeE.HOST:
+      return 'Host'
+    case RegistrationTypeE.PLATFORM:
+      return 'Platform'
+    default:
```

Comment:
There won't be a default here, because `type: RegistrationTypeE` can only be one of two registration types defined in the enum.

Reply:
Fixed